src/test/java/seedu/scheduler/logic/commands/ScheduleCommandTest.java

```diff
@@ -1,25 +1,114 @@
 package seedu.scheduler.logic.commands;
 
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.ThrowingSupplier;
 
+import javafx.collections.ObservableList;
+import seedu.scheduler.model.FilePath;
 import seedu.scheduler.model.Model;
 import seedu.scheduler.model.ModelManager;
-import seedu.scheduler.testutil.SampleInterviewee;
-import seedu.scheduler.testutil.SampleInterviewer;
-
+import seedu.scheduler.model.Schedule;
+import seedu.scheduler.model.person.Role;
 
 class ScheduleCommandTest {
+    private static final String INTERVIEWER = "interviewer";
+    private static final String INTERVIEWEE = "interviewee";
+
+    // ========================================= Hybrid Testing =================================================
     @Test
-    public void execute_sampleGraphData1() {
+    public void execute_allAllocated() {
+        Model model = importData("src/test/data/ScheduleTest/InterviewerTestData_1.csv",
+                "src/test/data/ScheduleTest/Interviewee_AllAllocated.csv");
+        CommandResult result = executeScheduleCommand(model);
+
+        // Test if message returned is correct
+        String expectedFeedback = "Successfully scheduled!\nAll interviewees are allocated with a slot!";
+        assertEquals(expectedFeedback, result.getFeedbackToUser());
+
+        // Test if underlying schedule structure is correct
+        String[] slotJohnDoeDetails = getSlotDescription("John Doe", model.getSchedulesList().get(0))
+                .get(0).split(",");
+        String[] slotJohnTohDetails = getSlotDescription("John Toh", model.getSchedulesList().get(2))
+                .get(0).split(",");
+
+        assertEquals("Department A", slotJohnDoeDetails[0]);
+        assertEquals("09/10/2019", slotJohnDoeDetails[1].trim());
+        assertEquals("18:30-19:00", slotJohnDoeDetails[2].trim());
+
+        assertEquals("Department B", slotJohnTohDetails[0]);
+        assertEquals("11/10/2019", slotJohnTohDetails[1].trim());
+        assertEquals("20:00-20:30", slotJohnTohDetails[2].trim());
+    }
+
+    // ========================================= Helper methods =================================================
+    /**
```

Comment:
Is it better to leave helper methods in the CommandTestUtil class? Theres a section for convenience methods in the class.